Source/WebCore/Modules/webxr/WebXROpaqueFramebuffer.cpp

```diff
@@ -212,117 +202,295 @@ void WebXROpaqueFramebuffer::endFrame()
     if (!gl)
         return;
 
+    gl->disableFoveation();
+
```

Comment:
Nit: May want to move the `disableFoveation()` call after the `WebXRLayerEndFrameStart` tracepoint so that call is included in the signpost interval.

Reply:
Should you do this?